packages/fxa-settings/src/styles/slider.css

```diff
@@ -36,11 +36,11 @@
 
 .AvatarCropper .MuiSlider-thumb {
   -webkit-appearance: none;
-  border: 1px solid #0060DF;
+  border: 1px solid #0060df;
```

Comment:
These tweaks were prettier's doing.

I'm going to file a follow up issue to take a look at this file and Tailwindify it if we can, I know we hit vendor/lib classes here and need the specificity but I think the properties could still adhere to our design guide 🤔